tabular/src/autogluon/tabular/trainer/model_presets/presets.py

```diff
@@ -78,6 +78,7 @@
     AG_TEXT_NN=TextPredictorModel,
     AG_IMAGE_NN=ImagePredictorModel,
     AG_AUTOMM=MultiModalPredictorModel,
+    FT_TRANSFORMER=FTTransformerModel,
```

Comment:
Is there a way for us to just reuse MultiModalPredictorModel but change the defaults to use FTTransformer?

Reply:
I think only the name is changed here. In [ft_transformer.py](https://github.com/awslabs/autogluon/pull/2085/files/9c1af3c527019794f0d42d2d147e316f10084b70#diff-e624d1064c00f87634e12d90d62d4d5c398d34cb5129f49adae2b22d2d0fdd5d), FTTransformer is implemented by changing the default params of MultiModalPredictorModel.